wps_tools/utils.py

```diff
@@ -113,13 +114,18 @@ def build_meta_link(
     return meta_link.xml
 
 
-def log_handler(process, response, message, process_step=None, level="INFO"):
+def log_file_path(process):  # From Finch bird
+    """Returns the filepath to write the process logfile."""
+    return Path(process.workdir) / "log.txt"
```

Comment:
I know this is a separate function in finch but I feel it would be alright to move this into the `log_handler` method.